modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -205,9 +205,10 @@
    (driver.common/start-of-week-offset driver)
    (partial hsql/call (u/qualified-name ::mod))))
 
-(def ^:private now (hsql/raw "SYSDATE"))
-
-(defmethod sql.qp/current-datetime-honeysql-form :oracle [_] now)
+(defmethod sql.qp/current-datetime-honeysql-form :oracle
+  [_]
+  (-> (hsql/raw "CURRENT_TIMESTAMP")
+      (hx/with-database-type-info "timestamp with time zone")))
```

Comment:
`SYSDATE` ignores the session timezone, `CURRENT_TIMESTAMP` doesn't.

Reply:
you should add this as a comment to the source here. I think that's a handy reminder that should stay in the code